### PR TITLE
fix: Remove some proptypes warning

### PR DESCRIPTION
**Pull request by Crash--**
we had a few errors. Let's remove them.

### CODE REVIEW

File: packages/cozy-procedures/src/components/Documents.jsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ class Documents extends React.Component {
             const filesStatusByCategory = filesStatus[categoryId]
             return (
               <section key={index}>
-                <Label>
+                <Label htmlFor="">
```
Comment (drazik) [inline packages/cozy-procedures/src/components/Documents.jsx]: I think this shows that `htmlFor` should not be a required prop on `Label`. What do you think?

Comment (Crash--) [reply packages/cozy-procedures/src/components/Documents.jsx]: Or that we shouldn't use `Label` since `label` should be use with `input`.

Comment (Crash--) [reply packages/cozy-procedures/src/components/Documents.jsx]: We should use `<Caption>` or something like that instead.

Comment (drazik) [reply packages/cozy-procedures/src/components/Documents.jsx]: OK :+1:

Comment (Crash--) [reply packages/cozy-procedures/src/components/Documents.jsx]: After reading the HTML spec, `for` is not mandatory, so let's remove the required

**Pull request merged**